crates/vertigo/src/computed/dependencies/refresh_edges.rs

```diff
@@ -1,7 +1,7 @@
 use std::collections::{BTreeMap, BTreeSet};
 use crate::computed::{Dependencies, GraphId, graph_value::GraphValueRefresh};
 
-enum RefreshState {
+pub enum RefreshState {
     Unknown,
```

Comment:
`Pending` or `CalculationPending` would be a better name

Reply:
Done